examples/spark2/src/main/scala/org/apache/carbondata/examples/CompareTest.scala

```diff
@@ -0,0 +1,372 @@
+          .option("tempCSV", "false")
+          .option("single_pass", "true")
+          .option("dictionary_exclude", "id") // id is high cardinality column
+          .mode(SaveMode.Overwrite)
+          .save()
+    }
+  }
+
+  // load data into parquet, carbonV2, carbonV3
+  private def prepareTable(spark: SparkSession): Unit = {
+    val df = generateDataFrame(spark)
+    println(s"loading ${df.count} records, schema: ${df.schema}")
+    val loadParquetTime = loadParquetTable(spark, df)
+    val loadCarbonV3Time = loadCarbonTable(spark, df, version = "3")
+    println(s"load completed, time: $loadParquetTime, $loadCarbonV3Time")
+    spark.read.parquet(parquetTableName).registerTempTable(parquetTableName)
+  }
+
+  // Run all queries for the specified table
+  private def runQueries(spark: SparkSession, tableName: String): Array[(Double, Array[Row])] = {
+    println(s"start running queries for $tableName...")
+    var result: Array[Row] = null
+    queries.zipWithIndex.map { case (query, index) =>
+      val sqlText = query.sqlText.replace("$table", tableName)
+      print(s"running query ${index + 1}: $sqlText ")
+      val rt = time {
+        result = spark.sql(sqlText).collect()
+      }
+      println(s"=> $rt sec")
+      (rt, result)
+    }
+  }
+
+  // run testcases and print comparison result
+  private def runTest(spark: SparkSession): Unit = {
+    val formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+    val date = new Date
+    val timestamp = date.getTime
+    // run queries on parquet and carbon
+    val parquetResult: Array[(Double, Array[Row])] = runQueries(spark, parquetTableName)
+    // do GC and sleep for some time before running next table
+    System.gc()
+    Thread.sleep(1000)
+    val carbonResult: Array[(Double, Array[Row])] = runQueries(spark, carbonTableName("3"))
+    // check result by comparing output from parquet and carbon
+    parquetResult.zipWithIndex.foreach { case (result, index) =>
+      if (result._2.length != carbonResult(index)._2.length) {
+        sys.error(s"result not matching for query ${index + 1}: " +
+            s"${result._2.length} and ${carbonResult(index)._2.length}")
+      }
+    }
+    // print all response time in JSON format, so that it can be analyzed later
+    queries.zipWithIndex.foreach { case (query, index) =>
+      println("{" +
+          s""""query":"${index + 1}", """ +
+          s""""parquetTime":${parquetResult(index)._1}, """ +
+          s""""carbonTime":${carbonResult(index)._1}, """ +
+          s""""fetched":${parquetResult(index)._2.length}, """ +
+          s""""type":"${query.queryType}", """ +
+          s""""desc":"${query.desc}",  """ +
+          s""""timestamp": "$timestamp" """ +
+          s""""date": "${formatter.format(date)}" """ +
+          "}"
+      )
+    }
+  }
+
+  def main(args: Array[String]): Unit = {
+    CarbonProperties.getInstance()
+        .addProperty("carbon.number.of.page.in.blocklet.column", "10")
```

Comment:
This property is remove now. so it is not required.
Now the size calculated in mb with property `carbon.blockletgroup.size.in.mb` and default size is 64. so no need to configure here.